client/src/components/Hit/Hit.js

```diff
@@ -0,0 +1,23 @@
+import { Highlight } from 'react-instantsearch-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Hit(props) {
```

Comment:
I think hits and props count as Algolia-domain specific, so JSdoc and some comments would be nice.